src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/GetUnallocatedCaseService.kt

```diff
@@ -37,6 +38,9 @@ class GetUnallocatedCaseService(
 ) {
 
   suspend fun getCase(crn: String, convictionNumber: Long): UnallocatedCaseDetails? {
+    if (laoService.getCrnRestrictions(crn).apopUserExcluded) {
+      throw NotAllowedForLAOException(crn, "A user of APoP is excluded from viewing this case")
```

Comment:
Are the arguments passed the wrong way round for this exception?

class NotAllowedForLAOException(msg: String, val crn: String) : RuntimeException(msg)

Reply:
good spot, thanks